src/index.js

```diff
@@ -1,10 +1,14 @@
 import _ from 'lodash';
 import './style.css';
+import result from './modules/results.js';
+import showResults from './modules/views.js';
 
 function component() {
   const element = document.createElement('div');
+  const listContainer = document.querySelector('.score');
+  listContainer.innerHTML = showResults(result);
 
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+  element.innerHTML = _.join(' ');
 
   return element;
 }
```

Comment:
- [ ] Great job so far your project looks great, :clap: however you should use arrow functions instead of the function keyword [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)